egg-herbie/main.rkt

```diff
@@ -22,7 +22,33 @@
                  [else "libegg_math"])
                 (bytes->string/utf-8 (system-type 'so-suffix)))))
 
-(define-ffi-definer define-eggmath (ffi-lib libeggmath-path))
+; Checks if Racket is being run in emulation via rosetta
+(define (check-for-rosetta)
+  (equal? (with-output-to-string (lambda () (system "sysctl -n sysctl.proc_translated"))) "1\n"))
+
+(define fallback-message
+  "Error: unable to load the 'egg-math' library.")

```

Comment:
This one starts with `Error:` but the one below doesn't. You should make them match up. Also, why not add a link to the bug tracker and say "Please file a bug here: <url>"?